infra/main.parameters.json

```diff
@@ -71,11 +71,29 @@
     "appServiceSkuName": {
       "value": "${AZURE_APP_SERVICE_SKU=B1}"
     },
+    "chatGptModelName":{
+      "value": "${AZURE_OPENAI_CHATGPT_MODEL_NAME}"
+    },
     "chatGptDeploymentName": {
-      "value": "${AZURE_OPENAI_CHATGPT_DEPLOYMENT=chat}"
+      "value": "${AZURE_OPENAI_CHATGPT_DEPLOYMENT_NAME}"
+    },
+    "chatGptDeploymentVersion":{
+      "value": "${AZURE_OPENAI_CHATGPT_DEPLOYMENT_VERSION}"
+    },
+    "chatGptDeploymentCapacity":{
+      "value": "${AZURE_OPENAI_CHATGPT_DEPLOYMENT_CAPACITY}"
+    },
+    "embeddingModelName":{
+      "value": "${AZURE_OPENAI_EMB_MODEL_NAME}"
     },
     "embeddingDeploymentName": {
-      "value": "${AZURE_OPENAI_EMB_DEPLOYMENT=embedding}"
+      "value": "${AZURE_OPENAI_EMB_DEPLOYMENT_NAME}"
```

Comment:
I prefer your name, but we try to keep azd env vars backwards compatible when possible, so that devs can use their existing envs.

Reply:
What do you mean?
I agree with you on backward compatibility but does that mean you want me to change these to the default stuff and the new names will follow the new standard to make them unique?

Reply:
We currently use:
AZURE_OPENAI_CHATGPT_DEPLOYMENT
AZURE_OPENAI_EMB_DEPLOYMENT

So those should stay the same, and the new ones look good.

Reply:
ok